login.php

```diff
@@ -59,7 +59,7 @@
 		<div class="navigation">
 			<!-- Set a div that contain logo for the music database -->
 			<div class="logo">
-				<a href = "#"><img src = "images/logo_test1.png" height = "100"></a>
+				<a href = "#"><img src = "images/logo_test2.png" height = "100"></a>
```

Comment:
20/9/2021: I create a second new logo. I will trialing to find the best logo(display in browser and get some feedback)